examples/nextjs-full-stack/README.md

```diff
@@ -60,6 +60,12 @@ Once your server is up and running:
 2. Open a board in your Developer team.
 3. To start the app, click the corresponding app icon in the app toolbar on the left.
 
+> ⚠️ In case of error `401` returned by a REST API request, remove the `miro_tokens` cookie for the `localhsot`
+> domain in the browser.
+>
```

Comment:
Is it possible to add either a screenshot / explanation of how to do that? I'm still not sure myself, so I think some devs maybe will not how to remove cookies for localhost specifically.

*typo in `localhsot`

Reply:
Is it 

Chrome -> Settings -> Privacy and Security -> Clear browsing data? I think this is not right.

Reply:
Well, it depends on the browser. For example, we can add a screenshot for Chrome's DevTools:

<img width="763" alt="image" src="https://github.com/miroapp/app-examples/assets/1961590/82da192e-6d29-432b-a740-ace29078bab3">

Reply:
Added

Reply:
looks great to me!

Reply:
Actually one small typo for `localhost` in line 63 @kirillsud

Reply:
![Screenshot 2023-09-14 at 13 31 38](https://github.com/miroapp/app-examples/assets/10428517/2665a6c0-adb7-46c7-9d7e-9067e8dc4190)

What is a bit strange is I dont have any cookies for localhost and I still see the 401 error

Reply:
sorry please ignore now I do see them